arboretum/permissions/checks/test_org_direct_collaborators.py

```diff
@@ -0,0 +1,115 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2021 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Github organization direct collaborators check."""
```

Comment:
This check isn't just for Github.  It should be applicable to Bitbucket and Gitlab once fetchers are implemented for those two hosting services.  We should also make the module more generic in case we want to add other checks related to collaborators (not just direct collaborators).  Suggest something like:

```suggestion
"""Repository organization/owner collaborators check."""
```

You should also rename the module file to something like `test_org_collaborators.py`